tests/function/gtEvent.cpp

```diff
@@ -290,7 +290,11 @@ TEST_F(LibopaecEventFCommonMOCKHW, event_drv_06) {
   poll_fd.events  = POLLIN | POLLPRI;
   poll_fd.revents = 0;
 
-  res = poll(&poll_fd, 1, 1000);
+  do
+  {
+    res = poll(&poll_fd, 1, 1000);
+    ASSERT_GE(res, 0);
+  } while(res == 0);
```

Comment:
Is there a chance of ever getting "stuck" here? Should we include a timeout period here or leave that up to gtest?

Reply:
Here I'm fixing the case where we get a spurious wake event.  If we get stuck, then we have a test failure.  I suppose we could implement a counter-based approach (max number of loops).